Feature/create atmosphere class

jonesbmj opened the pull request.

jonesbmj committed:
Edited .gitignore file to include .idea files

PyCharm adds weird .idea files somewhere/somehow and we don't want
them.

jonesbmj committed:
Added a basic version of the Atmosphere class

Includes a basic example. Later on, will increase complexity of each parameter to better model real world environment.

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,34 @@
+  viscosity: mu(h)
+  """
+
+  def __init__(self, altitude, density, gravity, pressure, temperature, viscosity):
```
Need a docstring for this function

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,34 @@
+
+
+# Example: constant, ideal atmosphere
+import numpy as np
```
Imports should be at the top of the file

ericdrejza commented on atmosphere.py:
```diff
@@ -0,0 +1,34 @@
+# Example: constant, ideal atmosphere
+import numpy as np
+
+h = np.arange(0, 100000, 1000)  # m         sea level to the Karman Line (edge of space)
```
Let's throw this is in a main function so you can execute it

def main():
\#This block of logic

if \_\_name\_\_ == '\_\_main\_\_':
main()

jonesbmj committed:
Addressed comments, added functions within class

Modified parameters of Atmosphere class. Added function for density,
gravity, pressure, temperature, viscosity. Used simple constants for the
time being. Added main() function.

jonesbmj committed:
Renamed functions for clarity.
